testcafe/12_home_profile.js

```diff
@@ -0,0 +1,157 @@
+import { Selector } from 'testcafe'
+
+import { getPathname } from './helpers/location'
+import { navigateToHomeAs } from './helpers/navigations'
+import { fetchSandbox } from './helpers/sandboxes'
+
+const HOME_URL = '/v2/accueil'
+
+fixture('En étant sur la nouvelle home,')
+
+test('j’ai accès au informations de mon profile', async t => {
+  const { user } = await fetchSandbox('pro_14_home_profile', 'get_pro_user')
+
+  const cardProfile = Selector('[data-testid="card-profile"]')
+  const userFirstname = Selector('span').withText(user.firstName)
+  const userLastname = Selector('span').withText(user.lastName)
+  const userEmail = Selector('span').withText(user.email)
+  const userPhoneNumber = Selector('span').withText(user.phoneNumber)
+
+  await navigateToHomeAs(user)(t)
+  await t
+    .expect(getPathname())
+    .eql(HOME_URL)
+    .expect(cardProfile.exists)
+    .eql(true)
+    .expect(userFirstname.exists)
+    .eql(true)
+    .expect(userLastname.exists)
+    .eql(true)
+    .expect(userEmail.exists)
+    .eql(true)
+    .expect(userPhoneNumber.exists)
+    .eql(true)
+})
+
+test("j’ai accès au formulaire d'édition de mon profile", async t => {
+  const { user } = await fetchSandbox('pro_14_home_profile', 'get_pro_user')
+
+  const modifyProfileButton = Selector('[data-testid="card-profile"] button').withText('Modifier')
+  // const profileDialog = Selector('[data-aria-labelledby="modal-profile"]')
+  const inputFirstname = Selector('input[name="first-name-input"]')
+  const inputLastname = Selector('input[name="last-name-input"]')
+  const inputEmail = Selector('input[name="email-input"]')
+  const inputPhoneNumber = Selector('input[name="phone-input"]')
+
+  await navigateToHomeAs(user)(t)
+  await t
+    .click(modifyProfileButton)
+    .expect(inputFirstname.exists)
+    .eql(true)
+    .expect(inputLastname.exists)
+    .eql(true)
+    .expect(inputEmail.exists)
+    .eql(true)
+    .expect(inputPhoneNumber.exists)
+    .eql(true)
+})
+
+test('je peu éditer mon profile avec de nouvelle données valides.', async t => {
```

Comment:
```suggestion
test('je peux modifier mon profil avec de nouvelles données valides.', async t => {
```
Voire juste "Je peux modifier mon profil"